docs/docs.css

```
@@ -102,6 +102,8 @@ summary {
   display: inline;
 }
 
+summary::-webkit-details-marker { display: none; }
```

```suggestion
summary::-webkit-details-marker {
    display: none;
}
```
The project uses this kind of styling for single entries.

---

I'm not picky :) This is fine!